src/views/AnalysisModule/Analysis/index.tsx

```diff
@@ -0,0 +1,89 @@
+import React from 'react';
+import { _cs } from '@togglecorp/fujs';
+
+import FormattedDate from '#rscv/FormattedDate';
+import ContainerCard from '#dui/ContainerCard';
+import Button from '#dui/Button';
+import Icon from '#rscg/Icon';
+import QuickActionButton from '#dui/QuickActionButton';
+
+import _ts from '#ts';
+
+import styles from './styles.scss';
+
+const emptyComponent = () => null;
+
+interface ComponentProps {
+    className?: string;
+    title: string;
+    startDate?: string;
+    endDate?: string;
+}
+
+function Analysis(props: ComponentProps) {
+    const {
+        title,
+        className,
+        startDate,
+        endDate,
+    } = props;
+
+    return (
+        <ContainerCard
+            className={_cs(className, styles.analysisItem)}
+            heading={title}
+            sub
+            headerDescription={(
+                <>
+                    <FormattedDate
+                        title={_ts('analysis', 'startDateLabel')}
+                        value={startDate}
+                        mode="MMM yyyy"
+                        emptyComponent={emptyComponent}
+                    />
+                    {startDate && endDate && (
+                        <div className={styles.separator}>-</div>
+                    )}
+                    <FormattedDate
+                        title={_ts('analysis', 'endDateLabel')}
+                        value={endDate}
+                        mode="MMM yyyy"
+                        emptyComponent={emptyComponent}
+                    />
+                </>
```

Comment:
Please create a re-usable component for this.

Reply:
Date Range? Makes sense. Will do.

Reply:
done